spec/expression.dd

```diff
@@ -514,8 +514,9 @@ $(GNAME ShiftExpression):
         $(D >)$(D >)$(D >) is an unsigned right shift.
     )
 
-    $(P It's illegal to shift by the same or more bits than the size of the
-        quantity being shifted:
+    $(P If the quantity is shifted by the same or more bits than the size of the
+	quantity, it is implementation defined whether a diagnostic is issued
```

Comment:
Then say what you mean. From what you say the implementation can still do whatever it wants. It may issue a diagnostic, or it may crash the computer. Let's be precise here. What is acceptable and what isn't?